src/routes/about-tss.svelte

```diff
@@ -1,35 +1,34 @@
 <script>
```

Comment:
No change on this file apart from a different image name - I formatted the file, it looks like a lot of changes!